src/post.js

```diff
@@ -0,0 +1,19 @@
+export default async function postScore(e) {
```

Comment:
- [ ] it would be better if you use the `Arrow function` instead of `function keyword` in order to practice what you learned and be familiar with both

Reply:
Hi @youmari Thanks for the kind review! Will implement the suggested change and resubmit. Hopefully should be approved.